components/usage/pkg/apiv1/usage_test.go

```diff
@@ -80,8 +80,9 @@ func newUsageService(t *testing.T, dbconn *gorm.DB) v1.UsageServiceClient {
 	)
 
 	costCenterManager := db.NewCostCenterManager(dbconn, db.DefaultSpendingLimit{
-		ForTeams: 0,
-		ForUsers: 500,
+		ForTeams:                  0,
+		ForUsers:                  500,
+		MinForIndividualsOnStripe: 1000,
```

Comment:
Nit: Slight cognitive dissonance between "Users" above and "Individuals" below (they're actually the same thing). Maybe the new field should be called `MinForUsersOnStripe` for consistency?

```suggestion
		ForUsers:                  500,
		MinForUsersOnStripe: 1000,
```